backend/internal/handlers/semesters/assignments/assignments.go

```diff
@@ -0,0 +1,132 @@
+package assignments
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/CamPlume1/khoury-classroom/internal/errs"
+	"github.com/CamPlume1/khoury-classroom/internal/models"
+	"github.com/gofiber/fiber/v2"
+)
+
+func (s *AssignmentsService) GetAssignmentsInSemester(c *fiber.Ctx) error {
+	classroomID, err := strconv.ParseInt(c.Params("classroomID"), 10, 64)
+	if err != nil {
+		return err
```

Comment:
What status code will this return? I think we should explicitly set them in handlers